crates/typst/src/layout/frame.rs

```diff
@@ -533,6 +533,40 @@ pub enum FrameItem {
     Tag(Tag),
 }
 
+impl FrameItem {
+    pub(in crate::layout) fn size(&self) -> Option<Size> {
+        match self {
+            FrameItem::Group(group) => {
+                Some(group.frame.size())
+            },
+            FrameItem::Text(text) => {
+                // FIXME: NEEDED
+                None
+            },
```

Comment:
Except for `FrameItem::Text` which doesn't store `Size` after `compose`. This seems like an omission to me because I'm assuming that data was present while `compose`ing `Frame` but not stored.

In a later commit I might end up adding this information like `FrameItem::Link`.